pluginserver.go

```diff
@@ -192,8 +192,8 @@ func getSchemaDict(t reflect.Type) schemaDict {
 		}
 		return schemaDict{
 			"type":   "map",
-			"keys":   schemaDict{"type": kType},
-			"values": schemaDict{"type": vType},
+			"keys":   kType,
```

Comment:
This change doesn't look correct at a first glance. Could you elaborate here @longquan0104? Perhaps elaborate on the error it fixes, with a stacktrace?